.travis.yml

```diff
@@ -12,16 +12,33 @@ compiler:
     - gcc
 
 env:
-    # Test with our internal pypi mirror
+    # Test with our internal pypi mirror, locale en-US
     - PIP_FIND_LINKS=http://pypi.pub.build.mozilla.org/pub
       PIP_NO_INDEX=1
       DISPLAY=$(if [[ $TRAVIS_OS_NAME = 'linux' ]]; then echo ':99'; fi)
       MOZ_XVFB=$(if [[ $TRAVIS_OS_NAME = 'linux' ]]; then echo 1; fi)
       PIP_STRICT_VERSIONS=--strict
-    # ... and with the official pypi website
+      LOCALE='en-US'
+
+    # ... and with the official pypi website, locale en-US
+    - DISPLAY=$(if [[ $TRAVIS_OS_NAME = 'linux' ]]; then echo ':99'; fi)
+      MOZ_XVFB=$(if [[ $TRAVIS_OS_NAME = 'linux' ]]; then echo 1; fi)
+      PIP_STRICT_VERSIONS=
+      LOCALE='en-US'
+
+    # Test with our internal pypi mirror, locale ru
+    - PIP_FIND_LINKS=http://pypi.pub.build.mozilla.org/pub
+      PIP_NO_INDEX=1
+      DISPLAY=$(if [[ $TRAVIS_OS_NAME = 'linux' ]]; then echo ':99'; fi)
+      MOZ_XVFB=$(if [[ $TRAVIS_OS_NAME = 'linux' ]]; then echo 1; fi)
+      PIP_STRICT_VERSIONS=--strict
+      LOCALE='.ru.'
```

Comment:
I think we can skip this complete block. Testing the internal PyPI should be enough with the en-US locale.